src/cli/commands/owner.js

```diff
@@ -87,14 +87,11 @@ async function list(config: Config, reporter: Reporter, flags: Object, args: Arr
   if (args.length > 1) {
     return false;
   }
-
   const name = await getName(args, config);
-
-  reporter.step(1, 3, reporter.lang('loggingIn'));
-  const revoke = await getToken(config, reporter, name);
-
-  reporter.step(2, 3, reporter.lang('ownerGetting', name));
-  const pkg = await config.registries.npm.request(name);
+  reporter.step(1, 1, reporter.lang('ownerGetting', name));
+  const pkg = await config.registries.npm.request(name, {
+    headers: {Accept: 'application/json'},
```

Comment:
Do we have to specify the headers? Isn't `config.registries.npm` doing it for us?

Reply:
I tried without headers and it didn't work, and such json headers appear in other commands also.

Reply:
@arcanis - maybe we can add this by default in a separate diff if it is not done already?